code/src/bip/datamodel/models.py

```diff
@@ -0,0 +1,221 @@
+import uuid as _uuid
+
+from django.core.validators import MinValueValidator, MaxValueValidator, \
+    RegexValidator
+from django.db import models
+
+from zds_schema.fields import RSINField
+from zds_schema.models import APIMixin
+from zds_schema.validators import (
+    UntilNowValidator, alphanumeric_excluding_diacritic,
+    validate_non_negative_string)
+
+from .constants import GeslachtsAanduiding, NaamgebruikAanduiding, \
+    AdresHerkomst, BurgerlijkeStaat, RedenOpschortingBijhouding, \
+    AanduidingBijzonderNederlandschap
+
+
+class IngeschrevenNatuurlijkPersoon(APIMixin, models.Model):
+    uuid = models.UUIDField(default=_uuid.uuid4)
+
+    burgerservicenummer = models.CharField(
+        null=True, blank=True, max_length=9,
+        validators=[validate_non_negative_string, ],
+        help_text='Het burgerservicenummer, bedoeld in artikel 1.1 van de Wet '
+                  'algemene bepalingen burgerservicenummer.'
+    )
+    a_nummer = models.BigIntegerField(
+        null=True
+    )
+    geslachtsaanduiding = models.CharField(
+        blank=True, max_length=1,
+        choices=GeslachtsAanduiding.choices,
+        help_text='Een aanduiding die aangeeft of de persoon een man of een '
+                  'vrouw is, of dat het geslacht nog onbekend is.'
+    )
+    burgerlijke_staat = models.CharField(
+        max_length=40, choices=BurgerlijkeStaat.choices
+    )
+    indicatie_geheim = models.CharField(
+        max_length=200 # TODO: choices
+    )
+    gemeente_van_inschrijving = models.IntegerField(
+        null=True
```

Comment:
nullable maar wel required? Integerfield vind ik ook raar voor een gemeente?

Reply:
Ja, ik ook. De exercitie in het fieldlab was echter de OAS uit Imvertor aanhouden. Alle obvious zaken heb ik opgelost (zoals een INP met max. 1 ouder), maar ze hebben hier geen resource voor.

Reply:
Mijn verwarring zit erin dat ik me niet echt kan voorstellen waar deze nummers dan vandaan komen, en of er wel/niet voorloopnullen zijn, maar het zal wel :-)